android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -762,7 +764,26 @@ public void setSelectedTextTrack(String type, Dynamic value) {
             trackIndex = value.asInt();
         } else { // default. invalid type or "system"
             trackSelector.clearSelectionOverrides(index);
```

Comment:
Do we need this or the setSelectionOverride anymore? If we are SDK <= 18 or no groups, the trackIndex will be C.INDEX_UNSET and we'll clear the overrides further down.

Reply:
@cobarx we need the setSelectionOverride in cases where captions were previously enabled, and are now explicitly disabled (or the system settings are disabled).  I've moved this up further in the code for clarity.